main.go

```diff
@@ -98,11 +104,15 @@ func main() {
 	listenAddr := "[::]:" + *listenPort
 
 	logger.Infof("🚀 Starting railtail")
-	logger.Infof("Mode                 : %s", mode)
-	logger.Infof("Listening on         : %s", listenAddr)
-	logger.Infof("Sending traffic to   : %s", *targetAddr)
-	logger.Infof("Tailscale hostname   : %s", *tsHostname)
-	logger.Infof("Tailscale state dir  : %s", tsStateDirPath)
+	logger.Infof("Mode                  : %s", mode)
+	logger.Infof("Listening on          : %s", listenAddr)
+	logger.Infof("Sending traffic to    : %s", *targetAddr)
+	logger.Infof("Tailscale hostname    : %s", *tsHostname)
+	logger.Infof("Tailscale state dir   : %s", tsStateDirPath)
+
+	if *tsLoginServer != "" {
+		logger.Infof("Tailscale Login server: %s", *tsLoginServer)
```

Comment:
Made this log conditional to not confuse users.

The default value `""` will be set to the default URL inside the `tsnet` package, so we don't have it available here.